src/main/java/carly/tasks/Event.java

```diff
@@ -0,0 +1,50 @@
+package carly.tasks;
+
+import java.text.MessageFormat;
+
+/**
+ * Represents a task of type Event.
+ * An Event task includes a description, start time, and end time.
+ */
+public class Event extends Task {
+    /** The task type identifier for Event tasks. */
+    private static final String TASK_TYPE = "E";
+
+    /** The end time of the event. */
+    private final String startTime;
+
+    /** The end time of the event. */
+    private final String endTime;
```

Comment:
Maybe you could copy paste some of the date time object methods from your deadline class to add this functionality for your event class as well.